packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,274 @@
+    if (this.props.isArrowDisplayed && this.props.contentAttachment !== 'middle center') {
+      this.offset = getContentOffset(this.attachment, this.props.targetRef(), PopupArrow.arrowSize, PopupContent.cornerSize);
+      arrow = <PopupArrow className={this.props.classNameArrow} refCallback={this.setArrowNode} />;
+      arrowPosition = primaryArrowPosition(this.attachment);
+      contentStyle = primaryMarginStyle(this.attachment, PopupContent.popupMargin);
+    }
+
+    return (
+      <PopupContent
+        {...boundsProps}
+        arrow={arrow}
+        arrowPosition={arrowPosition}
+        classNameInner={this.props.classNameContent}
+        closeOnEsc
+        closeOnOutsideClick
+        closeOnResize
+        isHeaderDisabled={this.props.isHeaderDisabled}
+        onRequestClose={this.props.onRequestClose}
+        refCallback={this.setContentNode}
+        style={contentStyle}
+      >
+        {this.props.children}
+      </PopupContent>
+    );
+  }
+
+  createPortalContent(content, useOverlay) {
+    if (!useOverlay) {
+      return content;
+    }
+
+    return (
+      <PopupOverlay className={this.props.classNameOverlay}>
+        {content}
+      </PopupOverlay>
+    );
+  }
+
+  render() {
+    const {
+      boundingRef,
+      children,
+      classNameArrow,
+      classNameContent,
+      classNameOverlay,
+      contentAttachment,
+      contentHeight,
+      contentWidth,
+      isArrowDisplayed,
+      isHeaderDisabled,
+      isOpen,
+      onRequestClose,
+      targetRef,
+      ...customProps
+    } = this.props; // eslint-disable-line no-unused-vars
+
+    let portalContent = children;
+    if (isOpen) {
+      let bidiContentAttachment = contentAttachment;
+      this.isRTL = document.getElementsByTagName('html')[0].getAttribute('dir') === 'rtl';
+      if (this.isRTL) {
+        bidiContentAttachment = switchAttachmentToRTL(bidiContentAttachment);
+      }
+
+      this.attachment = parseStringPair(bidiContentAttachment);
+
+      const boundingFrame = boundingRef ? boundingRef() : undefined;
+      const popupContent = this.createPopupContent(boundingFrame);
+      const allowScrolling = false;
+      const constraints = [
+        {
+          to: (boundingFrame || 'window'),
+          attachment: 'together',
+          pin: true,
+        },
+      ];
+
+      const tetherCotent = (
+        <TetherComponent
+          classPrefix="terra-Popup"
+          constraints={constraints}
+          content={popupContent}
+          contentAttachment={bidiContentAttachment}
+          contentOffset={`${this.offset.vertical} ${this.offset.horizontal}`}
+          isEnabled
+          onRepositioned={this.handleTetherRepositioned}
+          targetRef={targetRef}
+          targetAttachment={mirrorAttachment(bidiContentAttachment)}
+        />
+      );
+
+      portalContent = this.createPortalContent(tetherCotent, !allowScrolling);
+    }
+
+    return (
+      <Portal {...customProps} isOpened={isOpen}>
```

Comment:
Does adding customProps to the Portal potential cause the same issue as https://github.com/cerner/terra-core/issues/625?